Xamarin.Essentials/Connectivity/Connectivity.ios.cs

```diff
@@ -13,6 +14,26 @@ static void StartListeners()
             listener.ReachabilityChanged += OnConnectivityChanged;
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var doubleSignalStrength = new NEHotspotNetwork().SignalStrength;
```

Comment:
This allocation. I think it should be OK, but maybe wrap it in a using to dispose sooner?

Reply:
Yeah, I was worried about this as well. I was unable to decide whether to take the permanent memory hit and just lazy initialize a static NEHotspotNetwork which just sticks around forever once SignalStrength is used or allocate a new object on every call. This probably depends on the usage scenario as well, as if this were called in a rather tight loop or rather frequently would just add unnecessary memory pressure. If we decide to not cache the value we should definitely dispose the object ASAP to let i be destroyed on iOS.